polar-core/src/bindings.rs

```diff
@@ -301,17 +317,18 @@ impl BindingManager {
     }
 
     pub fn variable_state(&self, variable: &Symbol) -> VariableState {
-        self.variable_state_at_point(variable, self.bsp())
+        self.variable_state_at_point(variable, &self.bsp())
     }
 
-    pub fn variable_state_at_point(&self, variable: &Symbol, bsp: Bsp) -> VariableState {
+    pub fn variable_state_at_point(&self, variable: &Symbol, bsp: &Bsp) -> VariableState {
+        let bsp = bsp.bindings_index;
```

Comment:
GitHub swallowed my first attempt to say this:

nit: I think we should rename the local variables here & on line 541 from `bsp` to `index` to make it clear that it's an index instead of a full `Bsp` struct.

Reply:
good call